spec/models/profile_field_group_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   before do
     allow(FeatureFlag).to receive(:enabled?).with(:profile_admin).and_return(true)
+    allow(FeatureFlag).to receive(:enabled?).with(:profile_admin).and_return(true)
```

Comment:
why doing twice the same thing?

Reply:
Oops, this is my mistake, thanks for picking it up!